src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -152,52 +91,82 @@ def __init__(
         self._run_id = run_id
         self._variant_id = variant_id
         self._validate_inputs = validate_inputs
-        multiprocessing_start_method = os.environ.get("PF_BATCH_METHOD")
```

Comment:
Just a quick question

Did we expose "PF_BATCH_METHOD" to customers in anyways?

Do we need to add a logging here to say "PF_BATCH_METHOD" has been deprecated if we found it's set in ENV?

Reply:
Add logs to prompt users "The 'PF_BATCH_METHOD' has been deprecated"